httpbin/handlers_test.go

```diff
@@ -555,6 +557,54 @@ func testRequestWithBody(t *testing.T, verb, path string) {
 	}
 }
 
+func testRequestWithBodyBinaryBody(t *testing.T, verb string, path string) {
+	tests := []struct {
+		contentType  string
+		bodyResponse string
```

Comment:
Naming nit:  I think this should be called `requestBody`, since it's really defining the data we're sending in the request body.

Reply:
fixed